### PR TITLE
feat: add description to the application

### PR DESCRIPTION
**Pull request by baptistemesta**
Relates to [RUNTIME-366](https://bonitasoft.atlassian.net/browse/RUNTIME-366)

### CODE REVIEW

**Review by abirembaut — COMMENTED** [review]
Seems good, but maybe we should say a word about the error pages ?

**Comment by baptistemesta** [comment]
Yes. I guess we can put a lot more in the description:
Screenshot 2021-07-09 at 16 45 15

File: applications/bonita-application-directory.xml [hunk 1]
```diff
@@ -2,6 +2,7 @@
 <applications xmlns="http://documentation.bonitasoft.com/application-xml-schema/1.0">
     <application token="appDirectory" version="1.0" profile="_BONITA_INTERNAL_PROFILE_ALL" homePage="home" state="ACTIVATED" layout="custompage_layoutWithoutMenuBonitaV1" theme="custompage_themeBonita">
         <displayName>Bonita Application Directory</displayName>
+        <description>Choose which application to access from the list of available ones.</description>
```
Comment (abirembaut) [inline applications/bonita-application-directory.xml:5]:
```suggestion
        <description>This application allows to choose which application to access from the list of available ones and handle common server errors.</description>
```

File: bonita-distrib/bonita-application-directory.xml [hunk 2]
```diff
@@ -2,6 +2,7 @@
 <applications xmlns="http://documentation.bonitasoft.com/application-xml-schema/1.0">
     <application token="appDirectoryBonita" version="1.0" profile="_BONITA_INTERNAL_PROFILE_ALL" homePage="home" state="ACTIVATED" layout="custompage_layoutWithoutMenuBonita" theme="custompage_themeBonita">
         <displayName>Bonita Application Directory</displayName>
+        <description>Choose which application to access from the list of available ones.</description>
```
Comment (abirembaut) [inline bonita-distrib/bonita-application-directory.xml:5]:
```suggestion
        <description>This application allows to choose which application to access from the list of available ones and handle common server errors.</description>
```

**Review by abirembaut — COMMENTED** [review]
just a suggestion

**Comment by baptistemesta** [comment]
The feedbacks from @abirembaut was not integrated. I'll update it directly

**Pull request merged**